src/pages/shop/Shop.jsx

```diff
@@ -1,38 +1,45 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { SHOP_ITEMS, SECTIONS } from '../../constants';
 
-export const Shop = () => (
-  <div className="shop-container">
-    <div className="items-list-container">
-      <ul className="items-list-parent">
-        {SECTIONS.map((item) => (
-          <li key={item.id} className="items-list">{item.title}</li>
-        ))}
-      </ul>
-      <a href="https://www.google.com/" className="cart">
-        <i className="fas fa-shopping-cart fa-3x " />
-      </a>
-    </div>
-    <div className="items-container">
-      {SHOP_ITEMS.map((item) => (
-        <div className="item-container">
-          <div className="item-img-container">
-            <img className="item-img" src={item.img} alt={item.title} />
+export const Shop = () => {
+  const [currentsection, setSection] = useState(1);
```

Comment:
Add empty line.
Rename currentsection to currentSection, setCurrentSection
Get id from first sections array.